public/app/pages/ContinuousComparisonView.tsx

```diff
@@ -180,7 +180,9 @@ function ComparisonApp() {
           title={
             <ChartTitle
               titleKey={
-                isSidesHasSameUnits ? leftSide.metadata.units : undefined
+                isSidesHasSameUnits
+                  ? (leftSide.metadata.name as any)
```

Comment:
It's an easy escape hatch but more work is needed, so we can tackle this properly in the future.